src/components/jurisdiction/Jurisdiction.vue

```diff
@@ -67,6 +68,23 @@ export default class Jurisdiction extends Vue {
     return array
   }
 
+  /**
+   * Always exclude CA
+   * Exclude USA when states are listed in the jurisdiction list based on showUsJurisdictions flag
+   */
+  excludeJurisdictions (jurisdiction): Array<any> {
+    const excludedValues = []
+    if (jurisdiction === JurisdictionLocation.CA) {
+      excludedValues.push(JurisdictionLocation.CA)
+    }
+
+    if (this.showUsJurisdictions) {
+      excludedValues.push(JurisdictionLocation.US)
+    }
+
+    return excludedValues
+  }
```

Comment:
Whoa, this gets called for every item in the Intl list. And it does a lot of work.

Try something simpler.

Reply:
I think you only need a simple conditional here that returns False if `filter` should exclude it, else True.

Reply:
Looks good.